editor/code_editor.cpp

```diff
@@ -912,7 +912,7 @@ void CodeTextEditor::convert_case(CaseStyle p_case) {
 	for (int i = begin; i <= end; i++) {
 		int len = text_editor->get_line(i).length();
 		if (i == end)
-			len -= len - end_col;
+			len -= end_col;
```

Comment:
In https://github.com/godotengine/godot/pull/29306#discussion_r288936826 I was asking for review, not for a change. This changes the logic, did you test that it what it does now is the expected behaviour (and thus that the previous code was a bug)?

Reply:
Yeah, this was correct the first time, it should be `len = end_col;`